api/src/main/proto/stellarstation/api/v1/stellarstation.proto

```diff
@@ -0,0 +1,154 @@
+}
+
+// Request for the `OpenSatelliteStream` method.
+message SatelliteStreamRequest {
+  // The ID of the satellite to open a stream with. The ID of a satellite can
+  // be found on the StellarStation Console page for the satellite.
+  string satellite_id = 1;
+
+  // The `SatelliteStreamResponse.stream_id` from a previously opened stream to resume. If the
+  // specified stream has already expired or been closed, the stream is closed with an
+  // `INVALID_ARGUMENT` error.
+  string stream_id = 2;
+
+  // A request to send on the satellite stream. Currently only command sending to the satellite is
+  // supported. The request does not need to be filled on the first request to
+  // `OpenSatelliteStream`.
+  oneof Request {
+    // A request to send commands to the satellite.
+    SendSatelliteCommandsRequest send_satellite_commands_request = 3;
+  }
+}
+
+// A request to send commands to a satellite.
+message SendSatelliteCommandsRequest {
+  // The ID of the plan on which to send the commands.
+  //
+  // TODO(anuraag): Add reference to plans API after it is defined.
+  string plan_id = 1;
+
+  // The framing to be applied to `command` before transmission. If `command` has already been
+  // framed, this should be set to RAW and no additional processing will take place before
+  // modulation. When applying framing, each `command` will be treated as a single frame. Framing
+  // parameters used will be the ones registered for the satellite in the StellarStation Console.
+  Framing output_framing = 2;
+
+  // The command frames to send to the satellite. All commands will be transmitted in sequence
+  // immediately, during which time telemetry will not be received for the given `plan_id`. After
+  // all commands have been transmitted, telemetry receive will be immediately re-enabled. The
+  // maximum size of each command is 2MB. If a command larger than 2MB is received, the stream will
+  // be closed with a `RESOURCE_EXHAUSTED` error.
+  repeated bytes command = 3;
+}
+
+// A type of framing of a binary payload used in satellite communicaation.
+enum Framing {
+  // No framing. For sent payloads, this is usually pre-framed data while for received ones, it is
+  // raw IQ data.
+  RAW = 0;
+
+  // AX.25 (Amateur X.25) framing - https://en.wikipedia.org/wiki/AX.25
+  AX25 = 1;
+}
+
+// A response from the `OpenSatelliteStream` method.
+message SatelliteStreamResponse {
+  // The ID of this stream, which can be used in a subsequent call to `OpenSatelliteStream` to
+  // resume this stream on interruption (e.g., if a network failure causes the connection to be
+  // closed mid-stream). Telemetry received on a previously opened stream with no current listener
+  // will be buffered for a maximum of 1 minute, and the stream itself will be cancelled and no more
+  // buffering will occur if no client resumes the stream with `OpenSatelliteStream` within 10
+  // minutes.
+  string stream_id = 1;
```

Comment:
Could you add a comment if this id is unique among all customers or not since that information is important for Mission Control Software vendor who will handle multiple customers.

Reply:
All IDs are globally unique - I'd rather not mention that for every type of ID we have. Do you think we need to?